packages/mds-agency/request-handlers.ts

```diff
@@ -505,3 +525,23 @@ export const readStops = async (req: AgencyApiRequest, res: AgencyApiReadStopsRe
     return res.status(500).send({ error: new ServerError() })
   }
 }
+
+/* Experimental Handler */
+export const writeTripMetadata = async (
+  req: AgencyApiPostTripMetadataRequest,
+  res: AgencyApiPostTripMetadataResponse
+) => {
+  try {
+    const { provider_id } = res.locals
+    /* TODO Add better validation once trip metadata proposal is solidified */
+    const tripMetadata = validateTripMetadata({ ...req.body, provider_id })
+
+    await cache.writeTripMetadata(tripMetadata)
+    await stream.writeTripMetadata(tripMetadata)
```

Comment:
Any reason that these aren't concurrent operations via `Promise.all`?

Reply:
Nope, should be